lib/tasks/webpacker/check_binstubs.rake

```diff
@@ -1,7 +1,8 @@
 namespace :webpacker do
   desc "Verifies that bin/webpack & bin/webpack-dev-server are present."
   task :check_binstubs do
-    unless File.exist?("bin/webpack") && File.exist?("bin/webpack-dev-server")
+    unless Webpacker.config.custom_compile? ||
+      (File.exist?("bin/webpack") && File.exist?("bin/webpack-dev-server"))
```

Comment:
@dhh @gauravtiwari How do we work around this part for React on Rails users that will not want to have the bin/webpack file installed?

It turns out that if the automatic precompile task runs, this message prints:

`No compiling needed as everything is fresh`

as React on Rails first does the compile, so this seems OK.